pci.ids

```diff
@@ -1,225 +1,110 @@
-	04  PCI Hot-plug controller
-	05  SD Host controller
-	06  IOMMU
-	80  System peripheral
-	99  Timing Card
-# PTP Grandmaster Source Clock
-		01  TAP Timing Card
-C 09  Input device controller
-	00  Keyboard controller
-	01  Digitizer Pen
-	02  Mouse controller
-	03  Scanner controller
-	04  Gameport controller
-		00  Generic
-		10  Extended
-	80  Input device controller
-C 0a  Docking station
-	00  Generic Docking Station
-	80  Docking Station
-C 0b  Processor
-	00  386
-	01  486
-	02  Pentium
-	10  Alpha
-	20  Power PC
-	30  MIPS
-	40  Co-processor
-C 0c  Serial bus controller
-	00  FireWire (IEEE 1394)
-		00  Generic
-		10  OHCI
-	01  ACCESS Bus
-	02  SSA
-	03  USB controller
-		00  UHCI
-		10  OHCI
-		20  EHCI
-		30  XHCI
-		40  USB4 Host Interface
-		80  Unspecified
-		fe  USB Device
-	04  Fibre Channel
-	05  SMBus
-	06  InfiniBand
-	07  IPMI Interface
-		00  SMIC
-		01  KCS
-		02  BT (Block Transfer)
-	08  SERCOS interface
-	09  CANBUS
-C 0d  Wireless controller
-	00  IRDA controller
-	01  Consumer IR controller
-	10  RF controller
-	11  Bluetooth
-	12  Broadband
-	20  802.1a controller
-	21  802.1b controller
-	80  Wireless controller
-C 0e  Intelligent controller
-	00  I2O
-C 0f  Satellite communications controller
-	01  Satellite TV controller
-	02  Satellite audio communication controller
-	03  Satellite voice communication controller
-	04  Satellite data communication controller
-C 10  Encryption controller
-	00  Network and computing encryption device
-	10  Entertainment encryption device
-	80  Encryption controller
-C 11  Signal processing controller
-	00  DPIO module
-	01  Performance counters
-	10  Communication synchronizer
-	20  Signal processing management
-	80  Signal processing controller
-C 12  Processing accelerators
-	00  Processing accelerators
-# For the class of PCI attached devices which perform a function of Deep Learning Neural Network inference acceleration
-	01  AI Inference Accelerator
-C 13  Non-Essential Instrumentation
-C 40  Coprocessor
-C ff  Unassigned class
+# Syntax:
+# vendor  vendor_name
+#	device  device_name				<-- single tab
+#		subvendor subdevice  subsystem_name	<-- two tabs
+
+#ETH Cards
+
+
+1000  Broadcom / LSI
+	16d7  BCM57414 NetXtreme-E 10Gb/25Gb RDMA Ethernet Controller
+		14e4 1402  BCM957414A4142CC 10Gb/25Gb Ethernet PCIe
+
+8086  Intel Corporation
+        1533  I210 Gigabit Network Connection
+	1521  I350 Gigabit Network Connection
+        37d0  Ethernet Connection X722 for 10GbE SFP+
+	1572  Ethernet Controller X710 for 10GbE SFP+
+		1137 020a  Ethernet Converged NIC X710-DA2
+		1137 013b  Ethernet Converged NIC X710-DA4
+		1590 0000  Ethernet Controller X710 for 10GbE SFP+
+	1593  Ethernet Controller E810-C for SFP
+		1137 02be  E810XXVDA2 2x25/10 GbE SFP28 PCIe NIC
+
+15b3  Mellanox Technologies
+	1015  MT27710 Family [ConnectX-4 Lx]
+		15b3 0003  Stand-up ConnectX-4 Lx EN, 25GbE dual-port SFP28, PCIe3.0 x8, MCX4121A-ACAT
+	101d  MT2892 Family [ConnectX-6 Dx]
+1374  Silicom Ltd.

```

Comment:
Only vendor, without devices? What is this for in the "list of approved devices"?